### DISCUSSION TITLE
Docker poster path does not exist

### DISCUSSION THREAD

**Discussion by xela-nosretap** [meisnate12/Plex-Meta-Manager#866] [Q&A]
I had been running PMM directly on the machine, I then decided to change it to use the docker version of PMM.

I got everything up and running and copied the contents of the local config folder into the docker config folder.

Everything seems to be working correctly except for collection posters.

Here is an example of one of my collection entries:
```
Batman:
    sort_title: 'Batman 0000'
    tmdb_collection:
      - https://www.themoviedb.org/collection/120794
      - https://www.themoviedb.org/collection/263
    tmdb_movie: https://www.themoviedb.org/movie/414906-the-batman
    file_poster: config/assets/Batman.webp
```
when I look at the log file I see:
```
|=================================== Validating Batman Attributes ===================================|
|                                                                                                    |
| Validating Method: sort_title                                                                      |
| Value: Batman 0000                                                                                 |
|                                                                                                    |
| Validating Method: tmdb_collection                                                                 |
| Value: ['https://www.themoviedb.org/collection/120794', 'https://www.themoviedb.org/collection/263'] |
|                                                                                                    |
| Validating Method: tmdb_movie                                                                      |
| Value: https://www.themoviedb.org/movie/414906-the-batman                                          |
|                                                                                                    |
| Validating Method: file_poster                                                                     |
| Value: config/assets/Batman.webp                                                                   |
| Collection Error: Poster Path Does Not Exist: /run/s6/services/plex-meta-manager/config/assets/Batman.webp |
|                                                                                                    |
| Validation Successful                                                                              |
|                                                                                                    |
|==================================== Running Batman Collection =====================================|
```
In my `config.yml` my `asset_directory` is set to `config/assets`

Not sure if I need to change the `asset_directory` path to something else since it's now in docker.

**Comment by chazlarson** [comment]
Are you using the LSIO docker image?  The error seems like it, since that image uses a different current working directory than the official image or Python on the host:
```
Collection Error: Poster Path Does Not Exist: /run/s6/services/plex-meta-manager/config/assets/Batman.webp |
```
If you change to the official image chances are this just starts to work.

**Reply by xela-nosretap** [reply]
I assume LSIO is `linuxserver/plex-meta-manager:latest`? If so then I am.

**Reply by YozoraXCII** [reply]
> 
The Linuxserver.io image Is different to the official image in a few ways that cause problems.

One typical error is something like:
Path does not exist: /run/s6/services/plex-meta-manager/config/assets/SOMETHING


The result is that the stock config file and a lot of the examples found in the wiki and config repo don't work.

Generally, you should switch to using the official image, but if you use the LSIO image you should be aware of the following.

The LSIO image:

1. requires absolute paths in the config. /config/Movies.yml, not config/Movies.yml  This is the cause of the error above, and because of this most of the examples in the wiki and config repo don't work as-is with the lsio image.

2. only has latest, no develop or nightly.  If you want to switch to develop or nightly to try a new feature, you can't with LSIO image.

3. doesn't support runtime flags, only ENV vars.  This means that a command like:
docker run -it --rm -v /opt/pmm/config:/config linuxserver/plex-meta-manager --config config/config.yml -r --run-libraries "Movies - 4K DV"

doesn't work with the LSIO image; it would have to be:
docker run -it --rm -v /opt/pmm/config:/config -e PMM_CONFIG=/config/config.yml -e PMM_RUN=true -e PMM_LIBRARIES="Movies - 4K DV" linuxserver/plex-meta-manager


4. doesn't do manual runs correctly; they loop over and over.  With the official image, that command in the previous bullet point would run once and then quit, as expected.  The LSIO equivalent will run over and over until you manually kill the container.

5. Resets ownership of entire config dir every run.  In tests, the ownership of the config dir and its contents was set to 911:911 with each run.

Generally speaking, we suggest you use the official image instead of lsio.

**Reply by xela-nosretap** [reply]
Cool, that's good to know. I'll switch to the official image and see if that helps next time it runs.

That might explain why once I switched to the LSIO image, PMM went from running in 10-15 minutes to a few hours.

**Comment by meisnate12** [comment]
Also side note I'm not sure `.webp` images will work but it never hurts to try

**Reply by xela-nosretap** [reply]
I've been using `.webp` files for all my images for ages. Works with no issues and saves a lot of space.
